static/index.css

```
@@ -32,17 +32,30 @@ body {
   width: 100vw;
   height: 100vh;
 
-  background-color: var(--color-error-default);
+  background-color: var(--color-error-alternative);
   font-family: Euclid, Roboto, Helvetica, Arial, sans-serif;
 }
 
-p {
-  margin: 2em;
-}
-
 p a {
   text-decoration: underline;
-  color: var(--color-primary-default);
+  color: var(--color-error-inverse);
+  cursor: pointer;
+}
+
+ul {
+  padding-left: 2em;
+}
+
+button {
+  color: var(--color-error-alternative);
+  background: var(--color-error-inverse);
+  padding: 1em;
+  border-radius: 2em;
+  border: none;
+  float: right;
+  margin-top: 3em;
+  font-size: 0.9em;
+  font-family: Euclid, Roboto, Helvetica, Arial, sans-serif;
```

You can update the typography style to use the design token CSS variables for the typography too.

```suggestion
  color: var(--color-error-alternative);
  background: var(--color-error-inverse);
  padding: 1em;
  border-radius: 2em;
  border: none;
  float: right;
  margin-top: 3em;
  font-family: var(--typography-s-body-md-font-family);
  font-weight: var(--typography-s-body-md-font-weight);
  font-size: var(--typography-s-body-md-font-size);
  line-height: var(--typography-s-body-md-line-height);
  letter-spacing: var(--typography-s-body-md-letter-spacing);

  @media screen and (min-width: 768px) {
    font-family: var(--typography-l-body-md-font-family);
    font-weight: var(--typography-l-body-md-font-weight);
    font-size: var(--typography-l-body-md-font-size);
    line-height: var(--typography-l-body-md-line-height);
    letter-spacing: var(--typography-l-body-md-letter-spacing);
  }
```